src/index.html

```diff
@@ -1,13 +1,68 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+    rel="preconnect"
+    href="https://fonts.gstatic.com"
+    crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./main.css">
+  </head>
+  <body>
+
+    <div class="product-card" data-qa="card">
+      <img
+        src="./images/imac.jpeg"
+        alt="APPLE A1419 iMac 27' Retina 5K Monoblock (MNED2UA/A)"
+        class="product-card__img"
+      >
+
+      <h2 class="product-card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <p class="product-card__code">
+        Product code: 195434
+      </p>
+
+      <div class="product-card__rating">
+        <ul class="stars">
+          <li class="stars__star stars__star--inserted"></li>
+          <li class="stars__star stars__star--inserted"></li>
+          <li class="stars__star stars__star--inserted"></li>
+          <li class="stars__star stars__star--inserted"></li>
```

Comment:
You should specify active stars with modifier for `stars` BEM block, not with extra `inserted` class

Reply:
@Anna-Tkachenko sorry, but it's not better if I add element modifier 'stars__star--inserted' instead of I will use 'last-child' or something like that? I'm interested in this question, thanks.

Reply:
@vladyslav-lapshyn we should have one modifier for `stars` block with which we'll be able to control the number of active stars
For now, if we'll want to change number of active stars - we'll need to remove / add some classes from elements
If we'll have one modifier for `stars` block we'll just need to change one number: `stars--4` to `stars--5`

Reply:
@Anna-Tkachenko sorry, my mis, I understood it!

Reply:
> @vladyslav-lapshyn we should have one modifier for `stars` block with which we'll be able to control the number of active stars For now, if we'll want to change number of active stars - we'll need to remove / add some classes from elements If we'll have one modifier for `stars` block we'll just need to change one number: `stars--4` to `stars--5`

yea, to be honest when I was doing it I thought about change classes like in JS, as I said my mis) thank you for explain